### PR TITLE
mtx: add commands

### PR DESCRIPTION
**Pull request by DeadByElpy**
GetCurrentValues, GetEnergyCurrent, GetEnergyDay, GetExtendedCurrentValues

### CODE REVIEW

File: src/mtx/commands/downlink/GetCurrentValues.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    // returns full message - header with body
+    toBytes (): Uint8Array {
+        return new Uint8Array([COMMAND_ID, this.size]);
+    }
+}
+
+
+export default GetCurrentValues;
```
Comment (DarkPark) [inline src/mtx/commands/downlink/GetCurrentValues.ts:71]: no ending empty lines in all new files

File: src/mtx/commands/downlink/GetEnergyDay.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import GetEnergyDayResponse from '../uplink/GetEnergyDayResponse.js';
+
+
+const COMMAND_ID = 0x16;
+const COMMAND_SIZE = 3;
+
+
+const examples: TCommandExampleList = [
+    {
+        name: 'request day values for 2024.04.22 00:00:00 GMT',
```
Comment (DarkPark) [inline src/mtx/commands/downlink/GetEnergyDay.ts:17]:
```suggestion
        name: 'request day values for 2024.03.22 00:00:00 GMT',
```

?

File: src/mtx/commands/downlink/GetCurrentValues.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/mtx/commands/downlink/GetCurrentValues.ts:71]: they are present in files

File: src/mtx/commands/downlink/GetEnergyDay.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/mtx/commands/downlink/GetEnergyDay.ts:17]: fixed

**Review by Pnom — APPROVED** [review]

**Pull request merged**